inspector/an-inspector/ForgeModule/src/io/trigger/forge/android/modules/tabs/ModalView.java

```diff
@@ -252,6 +254,9 @@ public void run() {
                 view.addView(webView, new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT, 1));
                 webView.loadUrl(url);
 
+                // update content insets
```

Comment:
This comment makes no sense, it's the same text like the line below.

Reply:
I removed this in 489868f7ec198c76464d08526daa5f7d5efe9301